pages/extension/index.vue

```diff
@@ -5,7 +5,7 @@
 
       <div class="session-main">
         <Button
-          value="Connect Keplr"
+          value="Connect Lunie Extension"
```

Comment:
doesn't this need to work for both Keplr and Lunie extensions?

Reply:
Keplr has another section in CardSignInRequired

Reply:
CardSignInRequired isn't a component anymore. but ok, i see.